src/azure-cli/azure/cli/command_modules/reservations/_help.py

```diff
@@ -162,3 +162,84 @@
     type: string
     short-summary: Id of the subscription to look up applied reservations
 """
+
+helps['reservations reservation-order calculate'] = """
+type: command
+short-summary: calculate price
+long-summary: calculate price for reservation
+parameters:
+  - name: --applied-scope-type
+    type: string
+    short-summary: choose Single or Shared as appliedscope
+  - name: --applied-scopes
+    type: string
+    short-summary: Subscription that the benefit will be applied. Do not specify if --applied-scope-type is Shared.
+  - name: --billing-plan
+    type: string
+    short-summary: choose Monthly or Upfront
+  - name: --scope
+    type: string
+    short-summary: subscription that will be charged
+  - name: --display-name
+    type: string
+    short-summary: custom name
+  - name: --quantity
+    type: int
+    short-summary: quantity of product for calculating price
+  - name: --renew
+    short-summary: setup auto renewal
+  - name: --reserved-resource-properties
+    type: string
+    short-summary: Type of the Instance Flexibility to update the reservation with
```

Comment:
Please refine your help message and what is the reserved-resource-properties used for?

Reply:
updated the name and message